networking/hardware_networks/configuring-sriov-ib-attach.adoc

```diff
@@ -0,0 +1,17 @@
+[id="configuring-sriov-ib-attach"]
+= Configuring an SR-IOV InfiniBand network attachment
+include::modules/common-attributes.adoc[]
+:context: configuring-sriov-ib-attach
+
+toc::[]
+
+You can configure an InfiniBand (IB) network attachment for an Single Root I/O Virtualization (SR-IOV) device in the cluster. For the detailed object specification, refer to xref:../../networking/hardware_networks/configuring-sriov-ib-attach.adoc#nw-sriov-ibnetwork-object_configuring-sriov-ib-attach[InfiniBand device configuration object].
```

Comment:
```suggestion
You can configure an InfiniBand (IB) network attachment for an Single Root I/O Virtualization (SR-IOV) device in the cluster. For the detailed object specification, see xref:../../networking/hardware_networks/configuring-sriov-ib-attach.adoc#nw-sriov-ibnetwork-object_configuring-sriov-ib-attach[InfiniBand device configuration object].
```

Reply:
It's a little odd to me that you're xrefing to a module deeper in this assembly.

Reply:
@kalexand-rh, ideally I might have these as completely different topics, but without `xref` support both are compressed into the same topic. So this is my latest attempt to workaround not having access to `xref`. It would make even less sense to include the `xref` in an _Additional resources_ at the bottom of the page, I think.

But maybe it's clear enough from the table of contents that the object specification also lives on this page?